### DISCUSSION TITLE
Suggest to visitors who create an account on SILL that they declare the software they use when they register

### DISCUSSION THREAD

**Discussion by Sami-Hazgui** [codegouvfr/sill-web#470] [Ideas]
*   **Problème perçu:** Une fois le compte créé, il n'y a pas d'incitation à se déclarer utilisateur pour chaque logiciel qu'on utilise. En effet, créer un compte est suffisant pour avoir accès aux contacts de tous les référents du site.
*   **Contraintes pour régler ce problème:** C'est chronophage de se déclarer utilisateur pour chaque logiciel, de plus on ne comprend pas forcément l'utilité.
*   **Solution proposée :**

1) Ajouter une étape lors de la création du compte pour demander quels logiciels libres on utilise parmi les logiciels libres les plus populaires. Ajouter une barre de recherche pour en trouver d'autres.

2) donner accès aux adresses mails des référents d'un logiciel uniquement quand on s'est déclaré utilisateur de ce logiciel.

**Comment by guillermau** [comment]
Un contre argument que je pourrais donner pour la partie 2. de la proposition, ce serait le cas d'un utilisateur qui cherche à contacter un référent (pour des infos, conseils) pour commencer à utiliser un logiciel.
